bot/cogs/match.py

```diff
@@ -283,6 +300,24 @@ def all_ready(reaction, user):
             else:
                 raise ValueError(f'Team method "{team_method}" isn\'t valid')
 
+            voice_channel_one = self.bot.get_channel(697938536850128978)
+            await voice_channel_one.edit(name=f'Team {team_one[0].display_name}')
+            voice_channel_two = self.bot.get_channel(697938656752828416)
```

Comment:
These look like they're specific to your server.

Reply:
@b3none yes I just testing